core/arch/proc_api.h

```diff
@@ -312,59 +311,69 @@ typedef enum {
-    FEATURE_CRC32 = DEF_FEAT(AA64ISAR0, 4, 1, 0),    /**< CRC32<x> (AArch64) */
-    FEATURE_LSE = DEF_FEAT(AA64ISAR0, 5, 2, 0),      /**< Atomic instructions (AArch64) */
-    FEATURE_RDM = DEF_FEAT(AA64ISAR0, 7, 1, 0),      /**< SQRDMLAH,SQRDMLSH (AArch64) */
-    FEATURE_SHA3 = DEF_FEAT(AA64ISAR0, 8, 1, 0),     /**< EOR3,RAX1,XAR,BCAX (AArch64) */
-    FEATURE_SM3 = DEF_FEAT(AA64ISAR0, 9, 1, 0),      /**< SM3<x> (AArch64) */
-    FEATURE_SM4 = DEF_FEAT(AA64ISAR0, 10, 1, 0),     /**< SM4E, SM4EKEY (AArch64) */
-    FEATURE_DotProd = DEF_FEAT(AA64ISAR0, 11, 1, 0), /**< UDOT, SDOT (AArch64) */
-    FEATURE_FHM = DEF_FEAT(AA64ISAR0, 12, 1, 0),     /**< FMLAL, FMLSL (AArch64) */
-    FEATURE_FlagM = DEF_FEAT(AA64ISAR0, 13, 1, 0),   /**< CFINV,RMIF,SETF<x> (AArch64) */
-    FEATURE_FlagM2 = DEF_FEAT(AA64ISAR0, 13, 2, 0),  /**< AXFLAG, XAFLAG (AArch64) */
-    FEATURE_RNG = DEF_FEAT(AA64ISAR0, 15, 1, 0),     /**< RNDR, RNDRRS (AArch64) */
-    FEATURE_DPB = DEF_FEAT(AA64ISAR1, 0, 1, 0),      /**< DC CVAP (AArch64) */
-    FEATURE_DPB2 = DEF_FEAT(AA64ISAR1, 0, 2, 0),     /**< DC CVAP, DC CVADP (AArch64) */
-    FEATURE_JSCVT = DEF_FEAT(AA64ISAR1, 3, 1, 0),    /**< FJCVTZS (AArch64) */
-    FEATURE_FP16 = DEF_FEAT(AA64PFR0, 4, 1, 1),      /**< Half-precision FP (AArch64) */
-    FEATURE_RAS = DEF_FEAT(AA64PFR0, 7, 1, 0),       /**< RAS extension (AArch64) */
-    FEATURE_SVE = DEF_FEAT(AA64PFR0, 8, 1, 0),       /**< Scalable Vectors (AArch64) */
-    FEATURE_LOR = DEF_FEAT(AA64MMFR1, 4, 1, 0),    /**< Limited order regions (AArch64) */
-    FEATURE_SPE = DEF_FEAT(AA64DFR0, 8, 1, 0),     /**< Profiling extension (AArch64) */
-    FEATURE_PAUTH = DEF_FEAT(AA64ISAR1, 2, 1, 0),  /**< PAuth extension (AArch64) */
-    FEATURE_LRCPC = DEF_FEAT(AA64ISAR1, 5, 1, 0),  /**< LDAPR, LDAPRB, LDAPRH (AArch64) */
-    FEATURE_LRCPC2 = DEF_FEAT(AA64ISAR1, 5, 2, 0), /**< LDAPUR*, STLUR* (AArch64) */
-    FEATURE_FRINTTS = DEF_FEAT(AA64ISAR1, 8, 1, 0), /**< FRINT(32/64)(X/Z) (AArch64) */
-    FEATURE_BF16 = DEF_FEAT(AA64ZFR0, 5, 1, 0),     /**< SVE BFloat16 */
-    FEATURE_I8MM = DEF_FEAT(AA64ZFR0, 11, 1, 0),    /**< SVE Int8 matrix multiplication */
-    FEATURE_F64MM = DEF_FEAT(AA64ZFR0, 14, 1, 0),   /**< SVE FP64 matrix multiplication */
-    FEATURE_SVE2 = DEF_FEAT(AA64ZFR0, 0, 1, 0),     /**< Scalable vectors 2 (AArch64) */
-    FEATURE_SVEAES = DEF_FEAT(AA64ZFR0, 1, 1, 0),   /**< SVE2 + AES(AArch64) */
-    FEATURE_SVESHA3 = DEF_FEAT(AA64ZFR0, 8, 1, 0),  /**< SVE2 + SHA3(AArch64) */
-    FEATURE_SVESM4 = DEF_FEAT(AA64ZFR0, 10, 1, 0),  /**< SVE2 + SM4(AArch64) */
-    FEATURE_SVEBitPerm = DEF_FEAT(AA64ZFR0, 4, 1, 0), /**< SVE2 + BitPerm(AArch64) */
-    FEATURE_MTE = DEF_FEAT(AA64PFR1, 2, 1, 0),        /**< Memory Tagging Extension */
-    FEATURE_BTI = DEF_FEAT(AA64PFR1, 0, 1, 0),        /**< Branch Target Identification*/
+    /*                      Register NibblePos FeatVal Flags */
+    FEATURE_AESX = DEF_FEAT(AA64ISAR0, 1, 1, FEAT_GREATER_EQUAL),
```

Comment:
These should not lose their Doxygen commenting, e.g. `/**< AES<x> (AArch64) */`.

Reply:
Done